### PR TITLE
9145 Fix for the visual edit coordinates bug with multiple coordinates in a polygon (dev/7.4.x)

### PR DESCRIPTION
**Pull request by SDScandrettKint**
CSS changes to fix the edit coordinates bug.
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change
CSS fixes for the bug that caused the Edit Coordinates panel to be crammed with coordinates for a polygon with many coordinates.

### Issues Solved

#9145 

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: Knowledge Integration
*   Found by: @SDScandrettKint 
*   Tested by: @SDScandrettKint 
*   Designed by: @SDScandrettKint 

### Further comments

`map-coordinate-editor-crs-selector` change from fixed to absolute prevents "Coordinate preference" panel from moving down the screen when the overall scroll is used.
image

`map-coordinate-editor-header` change from fixed to absolute prevents "Edit Coordinates" panel from moving down the screen when the overall scroll is used.
image

`map-coordinate-editor-list` change fixes the ability to scroll through all coordinates and keeps the add new coordinate at the bottom when scrolling.

### CODE REVIEW

File: arches/app/media/css/arches.scss [hunk 1]
```diff
@@ -12857,8 +12857,10 @@ ul.select2-choices:after {
 }
 
 .map-coordinate-editor-list {
-    padding: 0 5px;
-    margin-top: 70px;
+    padding-top: 70px;
+    padding-left: 5px;
+    height: 250px;
```
Comment (chiatt) [inline arches/app/media/css/arches.scss:12862]: Could you increase the height to around 350px?

**Review by chiatt — CHANGES_REQUESTED** [review]
This is a nice improvement and fixes the bug. Eventually it would be nice to update the layout so that there isn't a need to hardcode the height. For the time being if you could increase the height of the `map-coordinate-editor-list`, then I think we can merge this in.

File: arches/app/media/css/arches.scss (hunk 1 above)
Comment (SDScandrettKint) [reply arches/app/media/css/arches.scss]: Resolved in the latest commmit [1d812e4](https://github.com/archesproject/arches/pull/10080/commits/1d812e43a6668f4f152484f3c7ed874ed638fc30)

**Comment by SDScandrettKint** [comment]
After changing the height in the latest commit, I've noticed that a height of 350px makes the "done" button in-line with the bottom of the container, and the final coordinate is cut in half, and the add new coordinate is missing:
image

When set to 250px, the UI looks as such:
image

I do agree that not hard coding the height would be the best for the future.

**Comment by chiatt** [comment]
I see - thanks for trying that @SDScandrettKint. Please change it back to 250px for the time being.

**Comment by SDScandrettKint** [comment]
After some further digging into this problem and fix, I've noticed that it negatively affects the "Map card" template which isn't affected by this bug and works well without the fix.  The CSS changes to the `map-coordinate-editor-list` which change the height are the reason.

On the "Map card" template, it is the `workbench-card-sidepanel-body` that has the scroll bar.  This allows it to scroll through existing coords, the add new X & Y option and the done button (which are all displayed on the `new-provisional-edit-card-container`).

The "Default card" template has a respective `workbench-card-sidepanel` but this has the overflow-y set to hidden.  I believe when I submitted my first PR for this (a year ago) I tried to add overflow scroll to this div but it is likely to be used in many other places and the overflow will have been set to hidden for a reason.  Adding overflow to `map-coordinate-editor-list` needs a height setting before it can be used to scroll, and of course it is used by the "Map card" template so changing height (and even the scroll) affects that template negatively.

A solution would be to add `style="overflow-y: scroll;"` to [line 33 of map.htm ](https://github.com/SDScandrettKint/arches-development/blob/ea5140503a0681b7b70996492268ec533a2e3e9a/arches/app/templates/views/components/map.htm#L33) so overflow is added for only the `map workbench-card-sidepanel-body`.  
With keeping the CSS changes to `map-coordinate-editor-crs-selector` and `map-coordinate-editor-header` in this PR, the final outputs would look like the following:

Map card template:
map_card_template_coords_scroll


Default card template:
default_card_template_coords_scroll


Some notable differences/issues:
- The coordinate reference doesn't stay on top for the default card like it does for the map card
- (Very minor) when selecting edit coordinates, the map card shows the top of the coordinates list first, but the default card shows the bottom of the list first (could be its own separate issue).

**Review by chiatt — APPROVED** [review]

**Pull request merged**